docs/aerodromes/sydney.md

```diff
@@ -0,0 +1,272 @@
+| >3000FT                | >5000M         | `EXPECT INDEPENDENT VISUAL APPROACH` |
+| Between 2000FT & 3000FT| >5000M         | `EXPECT INSTRUMENT APPROACH THEN INDEPENDENT VISUAL APPROACH WHEN VISUAL`|
+| Below 2000FT **or**    | <5000M         | `EXPECT INSTRUMENT APPROACH`                |
+
+#### Operational Info
+
+When parallel runways are used for departures but inbound traffic levels don't necessitate independent approaches (low traffic levels), the ATIS OPR INFO shall include:  
+`INDEPENDENT PARALLEL DEPARTURES IN PROGRESS`  
+
+When parallel runways are used for departures and the arrival sequence requires independent approaches (high traffic levels), the ATIS OPR INFO shall include:  
+`INDEPENDENT PARALLEL APPROACHES AND DEPARTURES IN PROGRESS`   
+
+When SODPROPS are in operation, the ATIS OPR INFO shall include:  
+`SIMULTANEOUS OPPOSITE DIRECTION PARALLEL RUNWAY OPERATIONS IN PROGRESS`  
+
+When Independent Visual Approaches are run during PROPS, the OPR INFO must also include:
+`DO NOT PASS THRU ASSIGNED RWY CENTRELINE`
+
+!!! example
+    <figure markdown>
+        ![IVAs on the ATIS](img/yssy_atis_example.png)
+    </figure>
+
+
+
+## Coordination
+### SY ADC / SY TCU
+#### Auto Release
+
+"Next" Coordination is a procedure where the **SY ADC** controller gives a heads-up to the SY TCU controller about an impending departure. The SY TCU controller will respond by assigning a heading to the aircraft, for the **SY ADC** controller to pass on with their takeoff clearance.
+
+!!! example
+    **SY ADC** -> **SY TCU**: "Next, ABC"  
+    **SY TCU** -> **SY ADC**: "ABC, Heading 030"  
+    **SY ADC** -> **SY TCU**: "Heading 030, ABC"  
+    **SY ADC** -> **ABC**: "ABC, Assigned heading right 030, Runway 34R, Cleared for Takeoff"  
+    **ABC** -> **SY ADC**: "Right heading 030, Runway 34R, Cleared for Takeoff, ABC"  
+    `AIP GEN 3.4`
+
+"Next" Coordination to SY TCU is additionally required for:  
+    a) Departures to YSBK  
+    b) Jets departing 16L via WOL, except during SODPROPS  
+    c) after a go-around, the next departure from that runway
+
+The SY TCU controller can suspend/resume Auto Release at any time, with the concurrence of **SY ADC**.
+
+"Next" Coordination to SY TCU is not required for aircraft assigned a **Procedural SID** and the Standard Assignable Level, or an aircraft assigned a **Radar SID** and assigned a standard assignable heading (see Coordination below).
+
+### SY ADC / SY SMC
+#### Ground Operations
+As per this (link) diagram, **SY ADC** has Responsibility of the Runways, meaning **SY SMC** must always coordinate with **SY ADC** to allow aircraft to cross runways whilst taxiing. **SY SMC** may request, or **SY ADC** may elect, to release certain runways to the **SY SMC** controller, so they may let aircraft cross the runway without coordination (For example, Releasing runway 07/25 to **SY SMC** whilst PROPS are in progress.) This Release may also be cancelled at the controller's discretion.
```

Comment:
missing link, or is it supposed to still be a placeholder?

Reply:
I didn't write this section, I assume it's a placeholder